tic-tac-toe/src/redux/Login/actions.js

```diff
@@ -1,29 +1,34 @@
 import authService from '@services/AuthService';
-import { push } from 'connected-react-router';
+import { JWTUSER } from '@constants/const';
 
 export const actions = {
-  CLICK_LOGIN: '@@LOGIN/CLICK_LOGIN',
-  AUTH: '@@LOGIN/AUTH',
-  ERROR: '@@LOGIN/ERROR',
-  CHANGE: '@@LOGIN/CHANGE'
+  LOGIN_LOADING: '@@LOGIN/CLICK_LOGIN',
+  LOGIN_SUCCESS: '@@LOGIN/AUTH',
+  LOGIN_FAILURE: '@@LOGIN/ERROR'
+};
+
+const serverError = 'An error occurred with the server';
+
+const privateActionCreators = {
+  loginSuccess: token => {
+    sessionStorage.setItem(JWTUSER, token);
+    return { type: actions.LOGIN_SUCCESS };
+  },
+  loginFailure: message => ({ type: actions.LOGIN_FAILURE, payload: message || serverError }),
+  change: state => ({ type: state ? actions.LOGIN_SUCCESS : actions.LOGIN_FAILURE })
```

Comment:
try not to use the same action for 2 diferent methods.. do you really need change action?